met-web/src/components/engagement/form/EngagementWidgets/WidgetTitle.tsx

```diff
@@ -24,12 +23,13 @@ export const WidgetTitle = ({ widget }: { widget: Widget }) => {
             return;
         }
         try {
+            if (title === '') setTitle(WidgetTitles[widget.widget_type_id]);
```

Comment:
Not requested but with how its implemented users can save an empty widget name. This will still allow them to have an empty textbox to type in but if they save it empty we will add a default widget name depending on the type. I thought it would be better I can definitely remove and let them save an empty name. What do you think